opt/cs50/bin/sqlite3

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+echo -en "\e[33m"
+if [[ "$#" -eq 0 ]]; then
+    echo "Warning: because you ran sqlite3 without any command-line arguments, no database will be opened or created by default. If you intended otherwise, quit sqlite3 with .quit and re-run sqlite3 with a command-line argument." | fold -s -w $(tput cols)
+elif [[ "$#" -eq 1 ]] && [[ "$1" == *.db ]] && [[ ! -f "$1" ]]; then
```

Comment:
Are we intentionally ignoring the case where we or the user may pass flags before/after the filename?

Reply:
Yeah, I looked at the man page, and it gets complicated quickly. Seems reasonable just to handle the two most common cases?